common/azure/src/main/kotlin/com/cosmotech/api/azure/adx/DeleteDataByScenarioRunId.kql

```diff
@@ -0,0 +1,3 @@
+.drop extents <|
```

Comment:
What's the purpose of this file ?

Reply:
To allow the user to delete the other tables updated with 'policy update'